src/utils/decodeHTML.ts

```diff
@@ -1,13 +1,13 @@
 /**
- * Image base64 data is currently stored with HTML encoding in the backend upon saving.
+ * Image base64 data, claim + Co is currently stored with HTML encoding in the backend upon saving.
```

Comment:
Is this suggestion true?

```suggestion
 * All data within the tenant service is currently stored with HTML encoding in the backend upon saving.
```

If so, maybe that would help to understand the scope of the problem?